best/model.py

```diff
@@ -355,7 +343,7 @@ def posterior_mode(self,
         float
             The posterior mode.
         """
-        samples = self.trace[var_name]
+        samples = arviz.extract(self.trace.posterior[var_name]).to_dataarray()
```

Comment:
This could be written more concisely as

```suggestion
        samples = arviz.extract(self.trace.posterior, var_names=var_name)
```